Tools/parametric_model/src/models/model_plots/aerodynamics_plots.py

```diff
@@ -194,46 +194,64 @@ def plot_liftdrag_curve(coef_dict, aerodynamics_dict):
     c_l_vec = np.zeros(aoa_deg.shape[0])
     c_d_vec = np.zeros(aoa_deg.shape[0])
 
-    if "c_l_wing_xz_offset" in coef_dict:
-        cl_0 = coef_dict["c_l_wing_xz_offset"]
-        cl_alpha = coef_dict["c_l_wing_xz_lin"]
-        cl_fp = coef_dict["c_l_wing_xz_fp"]
-        cd_0 = coef_dict["c_d_wing_xz_offset"]
-        cd_alpha = coef_dict["c_d_wing_xz_lin"]
-        cd_alpha2 = coef_dict["c_d_wing_xz_quad"]
-        cd_fp_min = coef_dict["c_d_wing_xz_fp_min"]
-        cd_fp_max = coef_dict["c_d_wing_xz_fp_max"]
+    if "cl0" in coef_dict:
+        cl_0 = coef_dict["cl0"]
+        cl_alpha = coef_dict["clalpha"]
+        cd_0 = coef_dict["cd0"]
+        cd_alpha = coef_dict["cdalpha"]
+        cd_alpha2 = coef_dict["cdalphasq"]
 
         stall_angle = aerodynamics_dict["stall_angle_deg"] * math.pi/180
-        sig_scale_fac = aerodynamics_dict["sig_scale_factor"]
 
         # region interpolation using a symmetric sigmoid function
         # 0 in linear/quadratic region, 1 in post-stall region
         for i in range(aoa_deg.shape[0]):
-            stall_region = cropped_sym_sigmoid(
-                aoa_rad[i], x_offset=stall_angle, scale_fac=sig_scale_fac)
-            # 1 in linear/quadratic region, 0 in post-stall region
-            flow_attached_region = 1 - stall_region
-            c_l_vec[i] = flow_attached_region * \
-                (cl_0 + cl_alpha * aoa_rad[i]) + \
-                stall_region * math.sin(2*aoa_rad[i])*cl_fp
-            c_d_vec[i] = flow_attached_region * \
-                (cd_0 + cd_alpha * aoa_rad[i] +
-                cd_alpha2 * aoa_rad[i] * aoa_rad[i]) + stall_region * \
-                (math.sin(aoa_rad[i])**2 * cd_fp_max +
-                (1-(math.sin(aoa_rad[i]))**2) * cd_fp_min)
+            c_l_vec[i] = (cl_0 + cl_alpha * aoa_rad[i])
```

Comment:
@Jaeyoung-Lim When looking at the test data results in the PR description, the slope for the linear approach seems to be very small, especially when compared to the global singularity-free model approach?

It might be possible that such an effect is caused by a value for cl_delta, which is far larger than it should be physically, but its effects are not visible in the lift plot? As this parameter is usually very small for fixed wing planes anyway, we could maybe even think about removing it from the estimation and check the results again. Otherwise, to include it in the lift coefficient plot, we would have to apply a zero-moment condition (or something comparable) to find a suitable delta_e to multiply cl_delta with, what would also be somewhat limiting...

Reply:
@sjschlapbach I think this we can think about, but the PR itself is not changing the behavior of the current implementation. Just visualizing as is

Reply:
Yes definitely, such a change would not really fit in here, probably. I just noticed that the disagreement between the two models seems to be very significant when looking at the plots, what might be confusing / indicating a wrong fitting in general, so I wanted to mention it :)

Reply:
@sjschlapbach Right, I agree with your comments. Also the global singularity free model should be defined in the zero lift axis, so there is no Cl_0 in the model itself. Lets try to address in the next PR :)